oneflow/ir/lib/OneFlow/PDLL/CMakeLists.txt

```
@@ -4,12 +4,15 @@ add_mlir_pdll_library(MLIROneFlowPDLLNormalizationPatternsIncGen NormalizationPa
                       NormalizationPatterns.h.inc)
 add_mlir_pdll_library(MLIROneFlowPDLLFuseConv2DBatchNormPatternIncGen
                       FuseConv2DBatchNormPattern.pdll FuseConv2DBatchNormPattern.h.inc)
+add_mlir_pdll_library(MLIROneFlowPDLLMulCastPatternIncGen MulCastPattern.pdll MulCastPattern.h.inc)                 
 oneflow_add_mlir_dialect_library(
   MLIROneFlowPDLLPatterns
+  MulCastPattern.cpp
```

```suggestion
  ElementwiseFusionPatterns.cpp
```
建议改名这个比较通用的名字，因为之后会添加更多